tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -16,6 +17,7 @@
 
 @parameterize_batch_for_data_sources(
     data_source_configs=[
+        MySQLDatasourceTestConfig(),
```

Comment:
TODO: actually make that list of data source test configs we talked about!

Reply:
Might be good to make the first ticket in the follow-up epic?